frontend/packages/dev-console/integration-tests/features/web-terminal/web-teminal-basic.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Web Terminal
+    As a user, I should be able to use web terminal
+
+
+Background:
+    Given user is at developer perspecitve
+    And user has installed Web Terminal operator
+    And user has created Web Terminal instance
+    And user has created namespace "aut-terminal-basic"
+    And user is at the Topolgy page
+
+
+@regression, @smoke
+Scenario: Web Terminal icon Masthead
```

Comment:
Remove this scenario, IMO this is not required

Reply:
Removed